### PR TITLE
Add Static Page

### PR DESCRIPTION
**Pull request by luisjesuszamora**
Add Static Page using Router and Material UI

### CODE REVIEW

File: client/src/components/Forum/StaticPage.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable jsx-a11y/anchor-is-valid */
```
Comment (luciancic) [inline client/src/components/Forum/StaticPage.js:2]: cleanup

File: client/src/components/Forum/Links.js [hunk 2]
```diff
@@ -67,6 +68,16 @@ export default function Links() {
...
         Guidlines
       </Link>
+      <div>
+      <Link variant="body2" href="#" onClick={preventDefault}>
+        Directives
+      </Link>
+      </div>
+      <div>
+        <Link component={RouterLink} to="/static-page">
+          Staic page using using react-router-dom TEST
```
Comment (luciancic) [inline client/src/components/Forum/Links.js:78]: Should be "Static Page"

File: client/src/components/Forum/StaticPage.js (hunk 1 above)
Comment (luciancic) [inline client/src/components/Forum/StaticPage.js:2]: cleanup

File: client/src/components/Forum/StaticPage.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+
+
+// notifications_none
+export default function StaticPage() {
+  
+  return (
+    <MyBox
+    // className={classes.root}
+    >
```
Comment (luciancic) [inline client/src/components/Forum/StaticPage.js:26]: `<MyBox>` would be better

**Comment by luisjesuszamora** [comment]
Hi Lucian,

After changed to branch add-static-page nothing worked for me. I mean I cannot start the website.
I have made a lot of things, like upading mongoose.

Thank you

[cid:dca1b67e-11e4-4ede-bd15-9b0e71d21e19]

________________________________
De : Lucian Condrea <notifications@github.com>
Envoyé : jeudi 23 juillet 2020 21:49
À : pdariusv/howdy-group-project <howdy-group-project@noreply.github.com>
Cc : luisjesuszamora <luiszamora123@hotmail.com>; Author <author@noreply.github.com>
Objet : Re: [pdariusv/howdy-group-project] Add Static Page (#17)


@luciancic commented on this pull request.

________________________________

In client/src/components/Forum/StaticPage.js<https://github.com/pdariusv/howdy-group-project/pull/17#discussion_r459747558>:

> +/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable jsx-a11y/anchor-is-valid */


cleanup

**Comment by luciancic** [comment]
Got it, @luisjesuszamora we can take a look at it together at the meet tonight

**Pull request merged**